index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="toolbar">
+        <div class="logo">
+            <h2>My logo</h2>
+        </div>
+        <div class="menu">
+            <img src="images/Union.png" alt="hamburger menu">
+        </div>
+    </div>
```

Comment:
- [ ] Please this should be a `section` tag not `div` stick to the design given by Microverse
![v](https://user-images.githubusercontent.com/61562626/129161800-88e9d767-6a69-4336-86dc-a0fce481c02b.jpeg)
.

Reply:
Okay, working on that